src/utils/index.ts

```diff
@@ -83,3 +83,28 @@ export function classNameFor(prefixes: string | string[], classSegments?: string
 
   return classnames(prefixList.map(p => [p].concat(classSegmentList).join("--")));
 }
+
+// This is defined as a separate function to avoid binding multiple event
+// listeners. We do not want this code to run more than once.
+function polyfillTouchAndHover() {
+  const { classList } = document.querySelector("body");
+
+  if ("ontouchstart" in window) {
+    classList.add("mq-touch-enabled");
+  } else {
+    classList.add("mq-hover");
+  }
+}
+
+/**
+ * Detect browser capabilities and apply faux media queries
+ *
+ * This will allow us to write CSS rules like:
+ *
+ *     .mq-hover .conditional-styles {
+ *       // only applied when the browser supports hover
+ *     }
+ */
+export function polyfillMediaQueries() {
```

Comment:
These functions were taken from Launchpad - the hover styles for the ResourceTile are dependent on the `.mq-hover` class being on the document body. Since `polyfillMediaQueries` must be run for this component to work, I think it makes sense to move the function into Dewey, although every repo would still be responsible for running this code on boot.

Reply:
Ah right, this code is also coming back to me. Having consumers use this function seems reasonable! That said, I wouldn't be surprised if someone forgot this and wondered why hover styles weren't working.

Maybe we should draw more attention to the comment in the docs?

```
// IMPORTANT! You need to call polyfillMediaQueries for hover styles to work.
```